internal/scheduler/testfixtures/testfixtures.go

```diff
@@ -343,15 +344,22 @@ func WithNodeAffinityJobs(nodeSelectorTerms []v1.NodeSelectorTerm, jobs []*jobdb
 }
 
 func WithRequestsJobs(rl schedulerobjects.ResourceList, jobs []*jobdb.Job) []*jobdb.Job {
```

Comment:
Proper deep copy, rather than mutating things that would be immutable if only go had the `readonly` keyword.